docs/driver_installation_zadig.md

```diff
@@ -44,3 +44,5 @@ Right-click it and hit **Uninstall device**. Make sure to tick **Delete the driv
 ![The Device Uninstall dialog, with the "delete driver" checkbox ticked](https://i.imgur.com/aEs2RuA.png)
 
 Click **Action → Scan for hardware changes**. At this point, you should be able to type again. Double check in Zadig that the keyboard device(s) are using the `HidUsb` driver. If so, you're all done, and your board should be functional again!
+
+?> Some users reported that they could fix their lenghty troubleshooting process by rebooting their computers.
```

Comment:
I'm not sure what you mean here by "lengthy troubleshooting process". Could you elaborate?

Reply:
I've seen it a couple of times in the help-channel with people who are new to QMK and who are overwhelmed by driver shenanigans and Zadig (just search for 'reboot' in #help). Although they follow the guide step by step in the end it comes down to windows needing a reboot, so I thought this guide should at least mention it once at the ende.

Reply:
Okay, how about this then:

```suggestion
?> A full reboot of your computer may sometimes be necessary at this point, to get Windows to pick up the new driver.
```

since this page is mainly using the second-person.

Reply:
Sounds good to me.